Description: Do not exclude "*/build/*" directory as buildd use that path
Author: Laurent Bigonville <bigon@debian.org>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/libssh/+bug/821437

--- a/doc/doxy.config.in
+++ b/doc/doxy.config.in
@@ -720,8 +720,7 @@
 
 EXCLUDE_PATTERNS       = */.git/* \
                          */.svn/* \
-                         */cmake/* \
-                         */build/*
+                         */cmake/*
 
 # The EXCLUDE_SYMBOLS tag can be used to specify one or more symbol names
 # (namespaces, classes, functions, etc.) that should be excluded from the
